pyproject.toml

```diff
@@ -0,0 +1,199 @@
+[build-system]
+requires = ['setuptools', 'wheel']
+build-backend = 'setuptools.build_meta'
+
+[project]
+authors = [{ name = 'DataCebo, Inc.', email = 'info@sdv.dev' }]
+classifiers = [
+    'Development Status :: 2 - Pre-Alpha',
+    'Intended Audience :: Developers',
+    'License :: Free for non-commercial use',
+    'Natural Language :: English',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Topic :: Scientific/Engineering :: Artificial Intelligence',
+]
+description = 'Benchmark tabular synthetic data generators using a variety of datasets'
+keywords = ['machine learning', 'synthetic data generation', 'benchmark', 'generative models']
+name = 'sdgym'
+version = '0.7.1.dev0'
+license = { text = 'BSL-1.1' }
+requires-python = '>=3.8,<3.12'
+readme = 'README.md'
+urls = { 'Homepage' = 'https://github.com/sdv-dev/SDGym' }
+
+dependencies = [
+    'appdirs>=1.3,<2',
+    'boto3>=1.15.0,<2',
+    'botocore>=1.18,<2',
+    'compress-pickle>=1.2.0,<3',
+    'humanfriendly>=8.2,<11',
+    "numpy>=1.20.0,<2;python_version<'3.10'",
+    "numpy>=1.23.3,<2;python_version>='3.10'",
+    "pandas>=1.1.3;python_version<'3.10'",
+    "pandas>=1.3.4;python_version>='3.10' and python_version<'3.11'",
+    "pandas>=1.5.0;python_version>='3.11'",
+    'psutil>=5.7,<6',
+    "scikit-learn>=0.24,<2;python_version<'3.10'",
+    "scikit-learn>=1.1.3,<2;python_version>='3.10'",
+    "scipy>=1.5.4,<2;python_version<'3.10'",
+    "scipy>=1.9.2,<2;python_version>='3.10'",
+    'tabulate>=0.8.3,<0.9',
+    "torch>=1.8.0;python_version<'3.10'",
+    "torch>=1.11.0;python_version>='3.10' and python_version<'3.11'",
+    "torch>=2.0.0;python_version>='3.11'",
+    'tqdm>=4.15,<5',
+    'XlsxWriter>=1.2.8,<4',
+    'rdt>=1.6.1,<2.0',
+    'sdmetrics>=0.11.0,<1.0',
+    'sdv>=1.3.0,<2',
+]
+
+[project.entry-points]
+sdgym = { main = 'sdgym.cli.__main__:main' }
+
+[project.optional-dependencies]
+
```

Comment:
I do not think you can have a blank line in TOML like this

Reply:
Yes, done in [7fac59d](https://github.com/sdv-dev/SDGym/pull/269/commits/7fac59d7ad872f32dc3b7307944cfe1980724f17)